### PR TITLE
[15.0][MIG] stock_demand_estimate

### PR DESCRIPTION
**Pull request by JoanMForgeFlow**
standard migration to v.15
@AaronHForgeFlow 
@mariadforgeflow
@bernatpforgeflow
@forgeflow

### CODE REVIEW

**Review by mariadforgeflow — APPROVED** [review]
Functional review OK.

**Review by BernatPForgeFlow — APPROVED** [review]
LGTM :+1:

File: stock_demand_estimate/__manifest__.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 {
     "name": "Stock Demand Estimate",
     "summary": "Allows to create demand estimates.",
-    "version": "14.0.1.1.0",
+    "version": "15.0.1.1.0",
```
Comment (LoisRForgeFlow) [inline stock_demand_estimate/__manifest__.py]: Usually during migrations, the version number is reset. In this case `15.0.1.0.0`

Comment (JoanMForgeFlow) [reply stock_demand_estimate/__manifest__.py]: Thanks! fixed

**Pull request merged**